asdf.sh

```diff
@@ -32,7 +32,7 @@ PATH="${ASDF_USER_SHIMS}:$PATH"
 # Load the asdf wrapper function
 . "${ASDF_DIR}/lib/asdf.sh"
 
-if [ -n "$ZSH_VERSION" ]; then
+if [ -n "${ZSH_VERSION-}" ]; then
```

Comment:
Can we use `:-` here for consistency with the above code

Reply:
No problem, done.